src/Dashboard.js

```diff
@@ -267,7 +267,7 @@ function Dashboard({ user }) {
                 }}
               >
                 <motion.div layoutId={`card-${item.id}`}>
-                {/* If a snapshot is available, display it /}
```

Comment:
The page doesn't actually load with these comments messed up.

Reply:
However, in a 'real job' you wouldn't be forking the repo, hence this being a new issue for me.